src/javascript/_common/base/client_base.js

```diff
@@ -432,18 +432,25 @@ const ClientBase = (() => {
     const isHighRisk = () => {
         const landing_companies = State.getResponse('landing_company');
         const risk_classification = State.getResponse('get_account_status.risk_classification');
+
         if (landing_companies) {
-            const financial_company_shortcode = landing_companies.financial_company.shortcode;
-            let gaming_company_shortcode;
+            let financial_company_shortcode, gaming_company_shortcode;
+            if (landing_companies.financial_company) {
+                financial_company_shortcode = landing_companies.financial_company.shortcode;
+            }
             if (landing_companies.gaming_company) {
                 gaming_company_shortcode = landing_companies.gaming_company.shortcode;
             }
+            const financial_restricted_countries = financial_company_shortcode === 'svg' && !gaming_company_shortcode;
+
+            const CFDs_restricted_countries = gaming_company_shortcode === 'svg' && !financial_company_shortcode;
+            
             const restricted_countries =
                 financial_company_shortcode === 'svg' ||
                 (gaming_company_shortcode === 'svg' && financial_company_shortcode !== 'maltainvest');
```

Comment:
not sure. left it in there in case there's a weird edge case

Reply:
should i remove?

Reply:
as far as i know there is no maltainvest shortcode for high/restricted countries. so your check on `restricted countries` for `financial_company_shortcode` on line 450 will always be true. and its the same as the `high_risk` variable below 🤔 
Please correct me if im wrong

Reply:
I think it covers different scenarios

Reply:
i see, then its safer to leave it there then 👀

Reply:
in that case, without 'maltainvest'

it would be restricted_countries = financial_company_shortcode === 'svg' || gaming_company_shortcode === 'svg'

and high_risk is restricted_countries = financial_company_shortcode === 'svg' && gaming_company_shortcode === 'svg'.

one is `&&` and the other is `||`